package.json

```diff
@@ -73,6 +73,7 @@
     "rollup-plugin-minify-html-literals": "^1.2.5",
     "rollup-plugin-serve": "^1.1.0",
     "rollup-plugin-terser": "^7.0.2",
+    "sass": "^1.49.9",
```

Comment:
After adding the sass tool you should be able to remove this from the devDependenies.

"wc-sass-render": "^1.2.3",